IntersectWith/src/IntersectWith.cpp

```diff
@@ -46,7 +45,13 @@ bool IntersectWith::local_work(msgpack::sbuffer *header, msgpack::sbuffer *paylo
 	assert(object);
 
 	bool gothit;
-	assert( object );
+
+    // if(pixel.depth >= world.maxdepth*2)
```

Comment:
The objects themselves can definitely tell when they're out of depth, but this was not an effective place to stop the flood.